Sources/Core/Internal/BundleImageLoader.swift

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+import CoreGraphics
+
+class BundleImageLoader: TextureImageLoader {
+    let bundle: Bundle
```

Comment:
This can be `private`, right?

Reply:
It always reminds me `Rule 4` of https://mattgemmell.com/api-design/, though I totally agree that it should be `private` here since it is an internal thing and we don't rely on accessing this param after the initialisation.

Reply:
Yeah, it also sends a clear message to other contributors not to access the bundle from this class, as doing so would kind of break the abstraction 👍